README.md

```diff
@@ -20,32 +20,21 @@ cri-tools aims to provide a series of debugging and validation tools for Kubelet
 - Building a new kubelet container runtime based on CRI.
 - Managing pods/containers for CRI-compatible runtimes by end-users, e.g. pods created by crictl may be removed automatically by kubelet because of non-exist on the kube-apiserver.
 
-## Current Status
-
-Version matrix:
-
-| Kubernetes Version | cri-tools Version | cri-tools branch |
-| ------------------ | ----------------- | ---------------- |
-| ≥ 1.16.x           | ≥ 1.16.x          | master           |
-| 1.15.X             | v1.15.0           | release-1.15     |
-| 1.14.X             | v1.14.0           | release-1.14     |
-| 1.13.X             | v1.13.0           | release-1.13     |
-| 1.12.X             | v1.12.0           | release-1.12     |
-| 1.11.X             | v1.11.1           | release-1.11     |
-| 1.10.X             | v1.0.0-beta.2     | release-1.10     |
-| 1.9.X              | v1.0.0-alpha.1    | release-1.9      |
-| 1.8.X              | v0.2              | release-1.8      |
-| 1.7.X              | v0.1              | release-1.7      |
-
-See the [roadmap](docs/roadmap.md) for information about current and future milestones.
-
-## Release Support Matrix
-
-The [releases](https://github.com/kubernetes-sigs/cri-tools/releases) are in tandem
-with Kubernetes releases in general. As referenced in above version matrix,
-since release `1.16`, we maintain `master` branch of this repo. However, for critical
-fixes in previous releases, we would only consider the releases which are still not EOL
-in Kubernetes releases.
+## Compatibility matrix: cri-tools ⬄ Kubernetes
+
+cri-tools follows the Kubernetes release cycles with respect to its minor
+versions (`1.x.y`). Patch releases (`1.x.z`) for cri-tools are not in sync with
+those from Kubernetes, because they are scheduled for each month, whereas
```

Comment:
this is kind of pedantic, but it caused me a moment of pause: the subject of this sentence is patch releases for cri-tools, but the second clause `because they are scheduled for each month` is referring to kubernetes releases. can you switch the order of cri-tools and kubernetes in the first clause?

Reply:
Fixed as suggested